scripts/build.js

```diff
@@ -24,25 +29,118 @@ const walk = function(dir) {
   return results
 }
 
-const allFiles = walk('./src/components')
-const allTwigFiles = allFiles.filter(file => file.substr(-5, 5) === '.twig')
-allTwigFiles.forEach(file => {
-  fs.copy(file, file.replace('./src/', './build/'))
-})
-
+// Compile dependencies
 const compiler = webpack(webpackConfig)
 compiler.run((err, stats) => {
   const report = stats.toJson()
   if (report.warnings.length) {
     report.warnings.forEach(message => console.warn(message))
   }
   if (report.errors.length) {
-    report.errors.forEach(message => console.error(message))
+    report.errors.forEach(message => {
+      throw new Error(message)
+    })
   } else if (err) {
-    console.error(err)
+    throw err
   } else {
-    console.timeEnd('Build completed in')
-    console.log(stats.hash, 'Done!')
+    createDistributionPackage(stats.hash).then(distStats => {
+      console.log(stats.hash, 'Done!')
+      createReport(distStats)
+    })
   }
 })
+
+const createReport = ({ hash, filename, size }) => {
+  const date = new Date()
+  const stats = {
+    hash,
+    date,
+    filename,
+    size
+  }
+  fs.writeFileSync(`${distFolder}/feed.json`, JSON.stringify(stats))
+
+  fs.writeFileSync(
+    `${distFolder}/index.html`,
+    `<!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Distribution ${date}</title>
+  </head>
+  <body>
+    <div class="stats">
+${Object.keys(stats)
+  .map(stat => {
+    return `      <div class="stat stat-${stat}">
+        <label style="font-weight:bold;">${stat}:</label>
+        ${stats[stat]}
+      </div>
+`
+  })
+  .join('')}    </div>
+    <a href="${filename}" title="download ${filename}" download>Download</a>
+  </body>
+  </html>`
```

Comment:
The HTML code can be indented a little more nicely here

Reply:
this is actually the correct way to indent for the output file. see `build/distribution/index.html`

Reply:
i've check `build/distribution/index.html` it looks 👌🏼